socs/agents/hwp_pmx/agent.py

```diff
@@ -235,10 +250,19 @@ def acq(self, session, params):
                 data['data']['current'] = curr
                 msg, volt = self.dev.meas_voltage()
                 data['data']['voltage'] = volt
+                msg, code = self.dev.check_error()
+                data['data']['err_code'] = code
+                data['data']['err_msg'] = msg
+                prot_code = self.dev.check_prot()
+                if prot_code != 0:
+                    self.prot = prot_code
```

Comment:
I'm not sure if I fully understand how to interpret the prot_code, but why isn't `self.prot` reset to 0 if `check_prot` returns 0?

Reply:
As a PMX specification, once the protection status code is read, the code is reset to 0 even though the protection mode is not actually cleared. The reason for this code is to continue to output the same error code unless clear_alarm() is done via the pmx agent.